### PR TITLE
Display membership years as actual ranges with gaps on member search

### PR DESCRIPTION
**Pull request by rlskoeser**

### CODE REVIEW

File: srcmedia/scss/members/_result.scss [hunk 1]
```diff
@@ -20,6 +20,18 @@
         margin-top: .5rem;
     }
 
+
+    .member-dates {
+        /* use padding to constrain size without messing up layout */
+        /* no more than 2 date ranges per line */
+        padding-left: calc(100% - 12em);
```
Comment (thatbudakguy) [inline srcmedia/scss/members/_result.scss:27]: any reason for using `em` here? don't think we do anywhere else

Comment (rlskoeser) [reply srcmedia/scss/members/_result.scss:27]: @thatbudakguy I was thinking of it in terms of characters on the line but forgot our local convention. I couldn't get a clear sense of how to size it in terms of space in the design but I could tell how many numbers we want. How do you suggest revising? Should I convert to rem?

Comment (thatbudakguy) [reply srcmedia/scss/members/_result.scss:27]: if it makes the most sense to think of it in terms of characters in the line, you could use the `ch` unit. converting to rem would work too.

Comment (rlskoeser) [reply srcmedia/scss/members/_result.scss:27]: @thatbudakguy did not know about `ch` unit! Which do you recommend?

Comment (thatbudakguy) [reply srcmedia/scss/members/_result.scss:27]: sorry, do you mean which unit? i think converting to rem for consistency is maybe better, but no strong preference.

**Pull request merged**